README.md

```diff
@@ -42,7 +41,6 @@ After you're finished please remove all the comments and instructions!
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
```

Comment:
- [ ] @Team Great work implementing the changes requested, however, you have not fixed the issue in the live demo link section, kindly consider removing the section or adding an actual link to the live demo, it's not good practice to have a wrong link in your documentation.

Reply:
I have already added N+1 problem when fetching all posts and their comments for a user. Please refer to the attached screenshots.
 
![N+1](https://github.com/umair-rao/New-Blog-App/assets/109624418/16f9b879-c763-482b-af13-ca0632523e0b)
![N+1 show](https://github.com/umair-rao/New-Blog-App/assets/109624418/e049b53e-d877-4b81-967c-d34bd4d0c3f3)